includes/cli/class-import.php

```diff
@@ -45,7 +48,16 @@ class Import extends WP_CLI_Command {
 	public function __invoke( $args, $assoc_args ) {
 		$ignore_terms = WP_CLI\Utils\get_flag_value( $assoc_args, 'ignore-terms', false );
 		$create_terms = WP_CLI\Utils\get_flag_value( $assoc_args, 'create-terms', false );
-		$file         = $args[0];
+		$ras_defaults = WP_CLI\Utils\get_flag_value( $assoc_args, 'ras-defaults', false );
+		$file         = $args[0] ?? null;
+
+		if ( ! $file && ! $ras_defaults ) {
+			WP_CLI::error( __( 'You must either inform a json file name or use the --ras-defaults flag.', 'newspack-popups' ) );
```

Comment:
Seems more appropriate:

```suggestion
			WP_CLI::error( __( 'You must either supply a json file path or use the --ras-defaults flag.', 'newspack-popups' ) );
```